packages/core/src/components/sections/Breadcrumb/Overrides.tsx

```diff
@@ -0,0 +1,27 @@
+import { Breadcrumb as UIBreadcrumb, Icon as UIIcon } from '@faststore/ui'
+
+import BreadcrumbCustomizations from 'src/customizations/components/overrides/Breadcrumb'
+
+const breadcrumbComponentsCustomization = {}
+
+const breadcrumbPropsCustomization = {} as any
+
+Object.entries(BreadcrumbCustomizations.components).forEach(([key, value]) => {
```

Comment:
This code repeats itself at every `Overrides.tsx` file. Should we move it somewhere?

Reply:
We will! It's part of the same task we'll be adjusting the types. It's the "Review Component Override API" task.